git_wrapper/branch.py

```diff
@@ -21,6 +22,12 @@ def __init__(self, git_repo, logger):
         self.git_repo = git_repo
         self.logger = logger
 
+    def _expand_file_path(self, path):
```

Comment:
This doesn't have to be now, but this seems generically useful for our projects, so it might make sense to move it to toolchest after this set of reviews.

Reply:
Agreed, it's pretty clear to me that'll move to a utils type module at some point, or the tool chest may make sense too if it's needed elsewhere. Just not rushing it until it's actually needed.